bluetooth: controller: Move calculation of max_tx_octets to ULL

mtpr-ot opened the pull request:
Move calculation of max_tx_octets from Nordic lll_conn.c to ULL, to
allow usage by other vendors and prevent duplicate code.

Signed-off-by: Morten Priess <mtpr@oticon.com>

cvinayak commented on subsys/bluetooth/controller/ll_sw/ull_conn.c:
```diff
@@ -1388,6 +1388,76 @@ u8_t ull_conn_llcp_req(void *conn)
 	return 0;
 }
 
+u16_t ull_conn_get_max_tx_octets(struct lll_conn *lll)
```
rename to ```ull_conn_lll_max_tx_octets_get```

mtpr-ot committed:
bluetooth: controller: Move calculation of max_tx_octets to ULL

Move calculation of max_tx_octets from Nordic lll_conn.c to ULL, to
allow usage by other vendors and prevent duplicate code.

Signed-off-by: Morten Priess <mtpr@oticon.com>

cvinayak reviewed (approved):
LGTM
